SignalServiceKit/src/Messages/Interactions/TSOutgoingMessage.m

```diff
@@ -39,6 +39,8 @@ @interface TSOutgoingMessage ()
 
 @property (atomic) TSGroupMetaMessage groupMetaMessage;
 
+@property (atomic) NSSet<NSString *> *readRecipientIds;
```

Comment:
This property gets reworked in the next PR to be a map of "recipient"-to-"read timestamp".